introduction.md

```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
 
 * Are computers and robots morally responsible for their actions? 
 
+### Autonomous Machines
+
+* Can software possess autonomy?
+
+* Is autonomy a subjective or objective quality? 
+
+* Is there a need to change our definition of autonomy?
+
+### Memetics: replication and curation
+
+* How can the intellectual framework of memetics be applied to understand modern information technology?
```

Comment:
Great argument. I agree that Memetics is suspicious and should therefore be left out. Let me rephrase my question: Since Luhman's "the medium is the message" it is generally acknowledged that technological evolution drives cultural evolution. However, is the reverse also true? How much does our cultural evolution feed back on our technological evolution? [to be expandend]